dynamics-gp/financials/BankReconciliation.md

```diff
@@ -962,7 +962,7 @@ The Deposit Totals window displays the number of checks, credit cards, and cash
 
 You entered a cash receipt total of \$117.65. The amount of \$117.00 appears in the Currency Total Amounts field and \$0.65 appears in the Coin Total Amounts field, as shown in the following table:
 
-|Number of      |   | Total amounts       |
+|Item      |Number of   | Total amounts       |
```

Comment:
I think this can be just "Number" instead of "Number of"

Reply:
Addressed.